PCbuild/prepare_ssl.py

```diff
@@ -1,4 +1,5 @@
 #! /usr/bin/env python3
+# -*- coding: utf8 -*-      for <2.7>/PC/VS9.0/build_ssl.py
```

Comment:
I opted for the Emacs/VI style mode-line to allow for trailing comment as to not have those editors complain.  If that is a non-issue for them, even better.